CHANGELOG.md

```diff
@@ -126,6 +126,10 @@
 
 * Make `empty_count` auto-correctable.  
   [KS1019](https://github.com/KS1019/)
+  
+* Make `private_swiftui_state` auto-correctable. Access control
+  has been made stricter and treats `fileprivate` as a violation.  
```

Comment:
I think that's not correct, it is? The rule still checks `modifiers.containsPrivateOrFileprivate()` and I'd better leave it that way (at least in this change as it's about auto-correction).

Reply:
You're correct, this was an artifact of my intermediate changes when I reverted back to the explicit rewriter approach. I'll update this so that it reflects the actual change.